openy.packages.yml

```diff
@@ -372,6 +372,7 @@ demo:
     - openy_demo_tfitness
     - openy_demo_tnews
     - openy_demo_webform
+    - y_lb_demo_content
```

Comment:
@podarok please add here a new submodule for the new demo content`  y_lb_init_demo_content`

Reply:
done